seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/sql/zeta/ZetaSQLFunction.java

```diff
@@ -665,4 +690,153 @@ private Object executeBinaryExpr(BinaryExpression binaryExpression, Object[] inp
+                    if (splitFieldValue == null) {
+                        continue;
+                    }
+                    if (splitFieldValue instanceof Object[]) {
+                        Object[] rowList = (Object[]) splitFieldValue;
+                        if (ArrayUtils.isEmpty(rowList) && isUsingOuter) {
+                            SeaTunnelRow outputRow = row.copy();
+                            outputRow.setField(fieldIndex, null);
+                            next.add(outputRow);
+                        } else {
+                            for (Object fieldValue : rowList) {
+                                SeaTunnelRow outputRow = row.copy();
+                                outputRow.setField(fieldIndex, fieldValue);
+                                next.add(outputRow);
+                            }
+                        }
+                    }
+                }
+                seaTunnelRows = next;
+            }
+            if (expression instanceof Function) {
+                Function function = (Function) expression;
+                String functionName = function.getName();
+                if (SPILT.equalsIgnoreCase(functionName)) {
+                    ExpressionList expressionList = function.getParameters();
+                    String column = ((Column) expressionList.get(0)).getColumnName();
+                    String delimiter = ((StringValue) expressionList.get(1)).getValue();
+                    List<SeaTunnelRow> next = new ArrayList<>();
+                    for (SeaTunnelRow row : seaTunnelRows) {
+
+                        int fieldIndex = outRowType.indexOf(column);
+                        Object splitFieldValue = row.getField(fieldIndex);
+                        if (splitFieldValue == null) {
+                            continue;
+                        }
+                        String[] splitFieldValues = splitFieldValue.toString().split(delimiter);
+                        for (String fieldValue : splitFieldValues) {
+                            SeaTunnelRow outputRow = row.copy();
+                            outputRow.setField(fieldIndex, fieldValue);
+                            next.add(outputRow);
+                        }
+                    }
+                    seaTunnelRows = next;
+
+                } else {
+                    throw new SeaTunnelRuntimeException(
+                            CommonErrorCodeDeprecated.UNSUPPORTED_OPERATION,
+                            "Transform config error! UnSupport function:" + functionName);
+                }
+            }
+        }
+        return seaTunnelRows;
+    }
+
+    public SeaTunnelRowType lateralViewMapping(
+            String[] fieldNames,
+            SeaTunnelDataType<?>[] seaTunnelDataTypes,
+            List<LateralView> lateralViews) {
+        if (CollectionUtils.isEmpty(lateralViews)) {
+            return new SeaTunnelRowType(fieldNames, seaTunnelDataTypes);
+        }
```

Comment:
I think this part should not including in `lateralViewMapping` method. Keep same as https://github.com/apache/seatunnel/pull/7928/files#diff-af2fe8799f25cd56d819141a2d8342e5ac0f2bc7e51220b8f4b3a2a0ada0c44dR241-R246